### PR TITLE
Added application backbone

### PR DESCRIPTION
**Pull request by evantimms**
If you guys have any questions let me know, this should hopefully be enough to build off of as this PR gets the main portion of our app going.

### CODE REVIEW

File: application/app/src/main/java/com/example/buber/Services/ApplicationService.java [hunk 1]
```diff
@@ -0,0 +1,5 @@
+package com.example.buber.Services;
+
+public class ApplicationService {
+    // TODO: Lukes code go here
```
Comment (evantimms) [inline application/app/src/main/java/com/example/buber/Services/ApplicationService.java:4]: @LukeDelaCruz Hey java wouldn't compile unless I have some class in the service package, so I made a base application service class but you are totally welcome to change it to however you want.

Comment (LukeDelaCruz) [reply application/app/src/main/java/com/example/buber/Services/ApplicationService.java:4]: Thanks. Mind fixing this build error? Not really sure what your intention is to override `getContactInfo` on Driver.

image

Comment (evantimms) [reply application/app/src/main/java/com/example/buber/Services/ApplicationService.java:4]: Yup

**Comment by evantimms** [comment]
Merging

**Pull request merged**